src/main/routes/logout.ts

```diff
@@ -2,14 +2,35 @@ import * as express from 'express'
 import * as config from 'config'
 import * as Cookies from 'cookies'
 
+import { IdamClient } from 'app/idam/idamClient'
+import { Logger } from '@hmcts/nodejs-logging'
+
 import { Paths } from 'app/paths'
+import { JwtExtractor } from 'idam/jwtExtractor'
+import { JwtUtils } from 'common/utils/jwtUtils'
+import { ErrorHandling } from 'common/errorHandling'
 
 const sessionCookie = config.get<string>('session.cookieName')
+const logger = Logger.getLogger('routes/logout')
 
 /* tslint:disable:no-default-export */
 export default express.Router()
-  .get(Paths.logoutReceiver.uri, (req: express.Request, res: express.Response, next: express.NextFunction) => {
-    const cookies = new Cookies(req, res)
-    cookies.set(sessionCookie, '', { sameSite: 'lax' })
-    res.redirect(Paths.homePage.uri)
-  })
+  .get(Paths.logoutReceiver.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response, next: express.NextFunction): Promise<void> => {
+      const jwt: string = JwtExtractor.extract(req)
+      const cookies = new Cookies(req, res)
```

Comment:
Please move it down, right before `cookies.set`. No need to declare it sooner and it is good to keep declaration closed to where it is needed.